views/new.ejs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+<h1>New Order Page</h1>
+
+	  <form action="/orders" method="POST">
+		    Order Number: <input type="text" name="orderNum"><br/>
+		   	Order Name: <input type="text" name="orderName"><br/>
+	    <input type="submit" name="" value="Submit Order">
+	  </form>
+
+	<button><a href="http://localhost:3000/orders/">home</a></button>
```

Comment:
in general, in an express app, use relative links for all your links: `<a href="/orders">`.... only leaving this comment here but it applies to all links